infrastructure/state.tf

```diff
@@ -2,12 +2,12 @@ terraform {
   backend "azurerm" {}
 
   required_providers {
-      azurerm = {
-          source  = "hashicorp/azurerm"
-          version = "3.109.0"
-      }
-      random = {
-          source = "hashicorp/random"
-      }
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.109.0"
     }
+    random = {
+      source = "hashicorp/random"
+    }
+  }
 }
```

Comment:
a new line?

Reply:
Autogenerated file added this, don't think this needs changing?